app/services/arcgis_api/geocoder.rb

```diff
@@ -0,0 +1,58 @@
+module ArcgisApi
+  class Geocoder
+    def suggest(text)
+      url = "#{root_url}/suggest"
+      params = {
+        text: text,
+        category: 'address',
+        countryCode: 'USA',
+        f: 'json',
+      }
+
+      parse_suggestions(
+        faraday.get(url, params) do |req|
+          req.options.context = { service_name: 'arcgis_geocoder_suggest' }
+        end.body,
+      )
+    end
+
+    def findAddressCandidates
```

Comment:
lower snake case this please
```suggestion
    def find_address_candidates
```

Reply:
Is there a change that's blocked by the inclusion of this method stub? If not, then I'd favor dropping it until we have an implementation.